base/shell/explorer/lang/nl-BE.rc

```diff
@@ -0,0 +1,200 @@
+STYLE DS_SETFONT | DS_MODALFRAME | DS_FIXEDSYS | DS_CONTEXTHELP | WS_POPUPWINDOW | WS_VISIBLE | WS_DLGFRAME
+CAPTION "Klassieke start menu aanpassen."
+FONT 8, "MS Shell Dlg"
+BEGIN
+    GROUPBOX         "Start menu", IDC_STATIC, 8, 7, 218, 114
+    ICON             IDI_STARTMENU2, IDC_STATIC, 19, 17, 20, 20
+    LTEXT            "U kan uw start menu aanpassen door onderdelen toe te voegen of te verwijderen.", IDC_STATIC, 48, 17, 108, 39
+    PUSHBUTTON       "Toevoegen...", IDC_CLASSICSTART_ADD, 162, 18, 53, 14
+    PUSHBUTTON       "&Verwijderen...", IDC_CLASSICSTART_REMOVE, 162, 33, 53, 14
+    PUSHBUTTON       "Aangepast", IDC_CLASSICSTART_ADVANCED, 162, 49, 53, 14
+    PUSHBUTTON       "&Sorteren", IDC_CLASSICSTART_SORT, 162, 65, 53, 14
+    ICON             IDI_RECYCLEBIN, IDC_STATIC, 20, 85, 20, 20
+    LTEXT            "Om recentelijke gebruikte documenten, programmas en web sitesuit het geheugen te wissen, druk op wissen", IDC_STATIC, 49, 85, 108, 33
+    PUSHBUTTON       "&Wissen", IDC_CLASSICSTART_CLEAR, 162, 94, 53, 14
+    LTEXT            "Geavanceerde start menu opties:", IDC_STATIC, 8, 129, 150, 8
+    CONTROL          "Start Menu instellingen", IDC_CLASSICSTART_SETTINGS, "SysTreeView32", WS_BORDER | 0x00000001, 8, 139, 218, 72, WS_EX_CLIENTEDGE
+    DEFPUSHBUTTON    "OK", IDOK, 122, 220, 50, 14
+    PUSHBUTTON       "Annuleren", IDCANCEL, 176, 220, 50, 14
+END
+
+IDD_FILENAME_WARNING DIALOGEX 0, 0, 272, 81, 0
+STYLE DS_SETFONT | DS_MODALFRAME | DS_FIXEDSYS | DS_CENTER | WS_POPUPWINDOW | WS_DLGFRAME
+CAPTION "Bestandsnaam waarschuwing"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    DEFPUSHBUTTON    "&Negeren", IDIGNORE, 215, 60, 50, 14
+    PUSHBUTTON       "&Aanpassen", IDC_FILE_RENAME, 159, 60, 50, 14
+    LTEXT            "", IDC_STATIC, 7, 51, 258, 1, NOT WS_GROUP
+    AUTOCHECKBOX     "&Deze check niet uitvoeren bij het opstarten.", IDC_NOTCHECK, 7, 62, 139, 10
+    ICON             32515, IDC_STATIC, 7, 7, 21, 20
+    LTEXT            "Er bestaat een bestand of folder op uw computer genaamd %s, dat ervoor zou kunnen zorgen dat bepaalde applicaties nit correct werken. De naam veranderen naar %s zou dit probleem oplossen. Wilt u de naam nu aanpassen?", IDC_STATIC, 34, 7, 224, 37
+END
+
+IDD_MODERNSTART_ADVANCED DIALOGEX 0, 0, 251, 198, 0
+STYLE DS_SETFONT | WS_CAPTION | WS_CHILD | WS_DISABLED
+CAPTION "Geavanceerd"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    GROUPBOX         "Start menu instellingen", IDC_STATIC, 7, 7, 238, 42
+    AUTOCHECKBOX     "&Submenus openen wanner ik mijn muis erboven houd", IDC_AUTOOPEN, 14, 21, 224, 10
+    AUTOCHECKBOX     "Nieuw geïnstalleerde programmas markeren.", IDC_HIGHLIGHT, 14, 35, 224, 10
+    LTEXT            "Start &menu onderdelen:", IDC_STATIC, 7, 53, 238, 8
+    CONTROL          "StartMenuInstellingen", IDC_ITEMS, "SysTreeView32", WS_BORDER | 0x00000001, 7, 63, 238, 67, WS_EX_CLIENTEDGE
+    GROUPBOX         "Recente Bestanden", IDC_STATIC, 7, 133, 238, 56
```

Comment:
Uppercase 'B'

Reply:
You mean, should it be in lowercase instead.